src/main/java/duke/Parser.java

```diff
@@ -26,10 +26,10 @@ public Command parse(String line) throws CommandParseException {
         int argWhitespace = line.indexOf(' ');
         if (argWhitespace == -1) {
             switch (line.toLowerCase()) {
-                case "bye":
-                    return new ByeCommand();
-                case "list":
-                    return new ListCommand();
+            case "bye":
+                return new ByeCommand();
+            case "list":
+                return new ListCommand();
```

Comment:
You might want to look into using Intellij's enhanced switch-case statements. For example:
```java
switch (itemCode) {
    case 001 -> System.out.println("It's a laptop!");
    case 002 -> System.out.println("It's a desktop!");
    case 003,004 -> System.out.println("It's a mobile phone!");
}
```